From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 20 Apr 2012 16:53:32 -0700
Subject: [025/106] staging: line6: toneport.c: remove err() usage

commit aa8f827a4d7dbea98a9076a4f85b1317391dc031 upstream.

err() was a very old USB-specific macro that I thought had
gone away.  This patch removes it from being used in the
driver and uses dev_err() instead.

CC: Markus Grabner <grabner@icg.tugraz.at>
CC: Stefan Hajnoczi <stefanha@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/line6/toneport.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.2.46/drivers/staging/line6/toneport.c
===================================================================
--- linux-3.2.46.orig/drivers/staging/line6/toneport.c	2013-07-26 19:28:24.000000000 +0000
+++ linux-3.2.46/drivers/staging/line6/toneport.c	2013-07-26 19:28:31.000000000 +0000
@@ -168,7 +168,7 @@
 			      cmd1, cmd2, NULL, 0, LINE6_TIMEOUT * HZ);
 
 	if (ret < 0) {
-		err("send failed (error %d)\n", ret);
+		dev_err(&usbdev->dev, "send failed (error %d)\n", ret);
 		return ret;
 	}
 
